.gitignore

```diff
@@ -67,3 +67,7 @@ GTAGS
 /.pants.rc
 /.venv
 .tool-versions
+# for rope refactoring
```

Comment:
This change is unrelated and should be proposed in a separate PR, especially since you can do this locally in your clone-specific `.git/info/exclude` file.

Reply:
Ok, makes sense, reverted

Thanks for `.git/info/exclude`, didn't know about it!